js/navigation.js

```diff
@@ -0,0 +1,35 @@
+//------------------
+//   Navigation Bar
+//------------------
+
+/* 
+1.) add a click event listener on hamburger image
+2.) when clicked nav style display is turned on || deleted (set to null?)
+3.) change the hamburger menu picture to display an X
+4.) have the event listener change
+*/
+
+fullScreenNav = document.querySelector('nav');
+hamburgerMenu = document.querySelector('.hamburger-menu');
+navigationBar = document.querySelector('.navigation');
+nav = document.querySelector('nav');
+function changeImage() {//Changes hamburger menu img to the close img
+    
+    //Don't judge me to hard on this
+    if (hamburgerMenu.src.includes('close')) {
```

Comment:
good job getting it done, but yeah, probably want to do this with CSS class for `display: none` rule and have two separate images